sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp

```diff
@@ -343,6 +343,41 @@ class group_sorter {
   }
```

Comment:
This comment is regarding https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp#L252

It's expected for `joint_sorter` (default and radix sorters both) to support oneDPL `zip_iterator` (https://github.com/intel/llvm/blob/sycl/sycl/include/sycl/ext/oneapi/experimental/group_helpers_sorters.hpp#L252 : Example 3).
Do you plan to add this support (no matter in this PR or a separate one)?

Reply:
I will look into this but not as part of this PR.

Reply:
ok. Thanks